Files/Views/SettingsPages/Preferences.xaml

```diff
@@ -114,6 +114,13 @@
                     Header="Show Library section on the sidebar"
                     HeaderTemplate="{StaticResource CustomHeaderStyle}"
                     IsOn="{Binding ShowLibrarySection, Mode=TwoWay}" />
+
+                <ToggleSwitch
+                    x:Name="OpenFoldersNewTab"
+                    x:Uid="SettingsPreferencesOpenFoldersNewTab"
+                    Header="Open Folders on new tab"
```

Comment:
```suggestion
                    x:Uid="SettingsPreferencesOpenFoldersNewTab"
                    Header="Open folders in new tab"
```

Can you also add this string to the en-US resource file?

Reply:
`Always open folders in new tab`

Reply:
@d2dyno1 Is that accurate? There is still the option to open folders in a new window and I don't want it to sound like that option will be ignored.

Reply:
I just wanted to suggest different name for it

```suggestion
                    x:Uid="SettingsPreferencesOpenFoldersNewTab"
                    Header="Always open folders in new tab"
```

Reply:
@d2dyno1 Using a different name is fine, I just don't think the one you suggested fits well in this case.